simulator-samples/pom.xml

```diff
@@ -33,7 +33,9 @@
     <module>sample-mail</module>
     <module>sample-combined</module>
     <module>sample-jms</module>
-    <module>sample-jms-fax</module>
+    
+    <!-- Removed because this sample is currently not functional -->
+    <!-- <module>sample-jms-fax</module> -->
```

Comment:
does this need some further investigation?

Reply:
[Fixed](https://github.com/citrusframework/citrus-simulator/pull/102/commits/34badcf783bf9a5d59479118c84e0bb6a00dd8ac)